### DISCUSSION TITLE
[Missing notification] non-existant data not notified

### DISCUSSION THREAD

**Discussion by grseabra** [rcasteran/jarvis4se#19] [Q&A] [answered]
When declaring the consumption of a **data for which the name is part of another data name**
--> JARVIS does not provide the notification "_<DATA_name>_ does not exist"

Exemple below:
SENSOR_TYPE does not existe at this time of the architecture definition
but STORED_SENSOR_TYPE exists

![image](https://user-images.githubusercontent.com/42869126/159887788-abc5e5a0-46e4-44d7-bae6-0137808c311a.png)

_[MINOR issue]_ No error on consumer declaration introduced in the .xml to the data STORED_SENSOR_TYPE

**Comment by Not2behere** [comment, marked as answer]
Hi Gregory, thanks for the report.
I'll open an issue for it and it will be corrected for next release (1.1.3).
